hs_core/hydroshare/resource.py

```diff
@@ -669,6 +669,8 @@ def add_resource_files(pk, *files, **kwargs):
 
     """
     resource = utils.get_resource_by_shortkey(pk)
+    if resource.raccess.published:
+        raise ValidationError("Can't add files to a published resource")
```

Comment:
This will block an admin from adding resource files which is different than what is possible today.  The same is true for delete files below.  @martinseul - is this ok?  I'm roping you in because there was a published resource you updated a file to as an admin last week.

Reply:
@sblack-usu in general we need a way to update files in under rare circumstances in the case you mentioned the db was corrupt and needed to be replaced, I'm OK with this restriction through the UI as it happens very seldom if there is a way for a developer to safely add and remove files through a command on the backend.

Reply:
I don't see a back door for this, we're going to have to add a check for admin